lib/stat_tracker.rb

```diff
@@ -12,110 +12,108 @@ def self.from_csv(file_paths)
   end
 
   def initialize(game_path, team_path, game_teams_path)
-    @game_path = game_path
-    @team_path = team_path
-    @game_teams_path = game_teams_path
+    @games = Games.new(game_path)
```

Comment:
This one is hard to explain but I had a mentor notice this that we dont need to instantiate the paths multiple times. Below on line 20 i have more notes on it.

Reply:
Bless be your mentor